stairctl/build.gradle

```diff
@@ -2,7 +2,8 @@ plugins {
     id 'application'
     id 'stairway.java-conventions'
 
-    id 'io.spring.dependency-management' version '1.1.4'
+    id 'io.spring.dependency-management' version '1.1.5'
+    // Spring Boot and Spring Shell versions should stay in sync
```

Comment:
Should or Must?

If Must, can we make this a variable instead of copy/pasting the same value?